src/main/java/com/project/service/ApplicationServiceImpl.java

```diff
@@ -41,14 +40,14 @@ public Application addApplication(Application application) throws UserException
 
     @Override
     public void deleteApplication(long id) throws ApplicationNotFoundException {
-        if(!applicationRepository.findById(id).isPresent())
+        if (!applicationRepository.findById(id).isPresent())
```

Comment:
`if` statements should be opened and closed with curly braces `{}` even if they have only one line of code.

Reply:
added curly braces {} for if statements in UserServiceImpl and ApplicationServiceImpl classes